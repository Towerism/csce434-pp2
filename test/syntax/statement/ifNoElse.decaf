int asdf() {
    if (e = 6) asdf;
}