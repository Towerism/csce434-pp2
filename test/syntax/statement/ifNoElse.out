
   Program: 
  1   FnDecl: 
         (return type) Type: int
  1      Identifier: asdf
         (body) StmtBlock: 
            IfStmt: 
  2            (test) AssignExpr: 
  2               FieldAccess: 
  2                  Identifier: e
  2               Operator: =
  2               IntConstant: 6
  2            (then) FieldAccess: 
  2               Identifier: asdf
